addons_rksv/pos_rksv/models/res_company.py

```diff
@@ -10,6 +10,7 @@ class Company(models.Model):
     _name = 'res.company'
     _inherit = 'res.company'
 
+    rksv_at = fields.Boolean('RKSV AT', default=True)
```

Comment:
Maybe we should make this a super user setting - it will be to difficult to control in source. For example - i have one customer with a normal store - and the same customer does have a club - where he also sells products - the club does not need rksv - the main store does need it.